### DISCUSSION TITLE
How do I change my fzf keybindings on an old version?

### DISCUSSION THREAD

**Discussion by Starcr0ss** [junegunn/fzf#3800] [Q&A]
I'm running fzf version 0.38.0 on Debian 12 (shipping old packages, per usual) and I want to change my keybinding for the fuzzy history search. Basically, I want fzf to allow me to search through my recent commands by simply pressing the up key instead of Control+T. How can I do this?

**Comment by LangLangBart** [comment]
>  I want fzf to allow me to search through my recent commands by simply pressing the up key 

You can list the ones you have:

```sh
bind -X | grep fzf
# "\C-r": "__fzf_history__"
# "\C-t": "fzf-file-widget"
```

Remove an assigned key
```sh
bind -r "\C-r"
```

Assign the new key, e.g. <kbd>↑</kbd>
```sh
bind -x '"\e[A" : "__fzf_history__"'
```

---

See also:
- [Bash Reference Manual - bind ](https://www.gnu.org/software/bash/manual/bash.html#index-bind)

https://github.com/junegunn/fzf/blob/01e7668915c4e3cf8c9eeca283d41beac924fe1f/shell/key-bindings.bash#L132-L137
